res/Sectors/M1105/Lorspane.txt

```diff
@@ -30,375 +30,374 @@
+0818                      D375312-8 Lo                           { -3 } (521+0) [1176] - -  - 600  9 Hv G5 V      
+0915                      A593201-E Lo                           {  2 } (B11-3) [244F] - KM - 913 13 Hv G6 V M6 V      
+0920                      E87A562-9 Ni Wa Cy O:1016              { -2 } (641-4) [5368] - -  - 900  6 Hv M7 V M6 V   
+1011 Sixaa                B6389T6-9 Hi (Sixaa)W                  {  2 } (D8D-2) [8B48] - -  - 214 13 Hv G6 V M6 V      
+1012                      B8987S4-5 Ag Pi                        {  1 } (367+1) [6865] - K  - 513 10 Hv M4 II M2 III      
+1016                      A6377S3-B                              {  2 } (565+4) [493C] - -  - 711  6 Hv A2 V K2 V G8 V      
+1018                      E7736S5-1 Ni                           { -3 } (450-1) [6314] - -  - 204 11 Hv M4 V       
+1112                      B844764-6 Ag Pi Sixa7 O:1011           {  1 } (369-2) [B8A8] - -  - 712 13 Hv M5 V M3 V       
+1113                      C531202-B Lo Po                        {  0 } (B11+3) [225D] - -  - 905  9 Hv G2 V M3 V      
+1116                      E491301-4 Lo He                        { -3 } (621+4) [1131] - -  - 404  8 Hv G0 V      
+1118                      E323413-6 Ni Po                        { -3 } (530+3) [1142] - -  - 600  8 Hv M6 II      
+1211                      C747663-5 Ag Ni Sixa7 Cy O:1011        { -1 } (B55+2) [8555] - S  - 302 10 Hv F1 V M6 V       
+1212                      C692661-3 He Ni Sixa9 Cy O:1011        { -2 } (750+1) [5413] - -  - 103 11 Hv M3 V G2 V      
+1213                      C451525-7 Ni Po                        { -2 } (642-1) [839A] - S  - 815 11 Hv M6 V M4 V       
+1219                      D100663-9 Na Ni Va Cy O:1519           { -2 } (A53+3) [347A] - -  - 214 10 Hv G2 IV      
+1317                      D4367S6-7                              { -2 } (465-1) [656B] - M  - 402 13 Hv M3 V M8 V      
+1319                      D635211-6 Lo                           { -3 } (611+3) [2159] - -  - 815 14 Hv G0 V M9 V      
+1320                      D753762-3 Po Cy O:1519                 { -2 } (A62+5) [3526] - M  - 603 10 Hv M1 V      
+1412                      C634104-9 Lo                           { -1 } (F01+1) [1148] - M  - 125 15 Hv G4 V      
+1413                      C200413-9 Ni Va                        { -1 } (E30+4) [535D] - -  - 405  8 Hv G8 V      
+1418                      C485405-6 Ni Pa                        { -2 } (B30+0) [82A6] - -  - 513 12 Hv M3 V K9 V      
+1420                      C321512-9 He Ni Po                     { -1 } (C42+3) [4479] - -  - 714 15 Hv G4 V      
+1513 Muadem               CAB4762-9 Fl Cy O:1518 (Muadem)        {  0 } (A66-3) [9727] - M  A 702  7 Hv M5 V K8 V      
+1515                      C7A38S4-9 Fl Ph                        {  0 } (87A+2) [B878] - -  - 201  9 Hv K3 V M7 V M5 V      
+1518 Etribb               A100301-G Lo Va                        {  2 } (D21-2) [457F] - -  - 712 12 Hv F9 V M5 V      
```

Comment:
Per the T5 Second Survey rules this TL should be reduced to F or E

Reply:
Updated